examples/apps/presence-tracker/src/FocusTracker.ts

```diff
@@ -3,120 +3,74 @@
  * Licensed under the MIT License.
  */
 
-import { ISignaler } from "@fluid-experimental/data-objects";
 import { TypedEventEmitter } from "@fluid-internal/client-utils";
-import type { IAzureAudience } from "@fluidframework/azure-client";
-import { IContainer } from "@fluidframework/container-definitions/internal";
 import { IEvent } from "@fluidframework/core-interfaces";
-import { IMember } from "fluid-framework";
+import type {
+	IPresence,
+	ISessionClient,
+	LatestValueManager,
+	PresenceStates,
+} from "@fluidframework/presence/alpha";
+import { Latest, SessionClientStatus } from "@fluidframework/presence/alpha";
 
 export interface IFocusTrackerEvents extends IEvent {
-	(event: "focusChanged", listener: () => void): void;
+	(event: "focusChanged", listener: (focusState: IFocusState) => void): void;
 }
 
-export interface IFocusSignalPayload {
-	userId: string;
-	focus: boolean;
+export interface IFocusState {
+	hasFocus: boolean;
 }
 
 export class FocusTracker extends TypedEventEmitter<IFocusTrackerEvents> {
-	private static readonly focusSignalType = "changedFocus";
-	private static readonly focusRequestType = "focusRequest";
-
-	/**
-	 * Local map of focus status for clients
-	 *
-	 * @example
-	 *
-	 * ```typescript
-	 * Map<userId, Map<clientid, hasFocus>>
-	 * ```
-	 */
-	private readonly focusMap = new Map<string, Map<string, boolean>>();
-
-	private readonly onFocusSignalFn = (clientId: string, payload: IFocusSignalPayload) => {
-		const userId: string = payload.userId;
-		const hasFocus: boolean = payload.focus;
-
-		let clientIdMap = this.focusMap.get(userId);
-		if (clientIdMap === undefined) {
-			clientIdMap = new Map<string, boolean>();
-			this.focusMap.set(userId, clientIdMap);
-		}
-		clientIdMap.set(clientId, hasFocus);
-		this.emit("focusChanged");
-	};
+	private readonly focus: LatestValueManager<IFocusState>;
 
 	constructor(
-		container: IContainer,
-		public readonly audience: IAzureAudience,
-		private readonly signaler: ISignaler,
+		private readonly presence: IPresence,
+		// eslint-disable-next-line @typescript-eslint/ban-types
```

Comment:
Nit: can we add a quick comment explaining why we needed to disable the eslint rule here?